README.md

```diff
@@ -15,22 +15,19 @@ Booster is a pre-release framework under heavy development, so non-backward-comp
-present some important challenges (mainly related to complexity).
-
-If a genie gave us three Serverless wishes, we would ask for:
-
-- Smoother learning curves beyond the getting-started guide.
-- Easy-to-use conventions and standards for common recurrent problems.
-- Higher-level abstractions that make it easier to represent business rules in the cloud.
-
-Booster provides a set of highly opinionated conventions and sensible defaults to automatically build and provision the
-cloud architecture needed to run your projects without asking you to write any configuration at all.
-In fact, the configuration is inferred _automatically_ from the code you write.
+* *Play nicely*: Booster is not here to replace your toolkit, but to expand it. Booster's goal is to come along well together with your existing auth, queues, databases and services, providing a modern and swift tool to build new functionality that really squeezes the cloud.
+* *DDD:* Software should be designed around business-level concepts to ease team's communication. All code in Booster is defined on terms of Commands, Events, Handlers and Entities, limiting the need of artificial developers-only constructs.
+* *CQRS and Event-Sourcing:* Booster is designed around the concepts of CQRS and Event-Sourcing. This design has many advantages regarding scalability and data management. It even allows you to travel back in time!
+* *The cloud is the machine:* We believe that infrastructure should be created transparently in the same way that a compiler hides the details of the target processor. We often think about Booster as the "TypeScript-to-Cloud compiler".
+* *True Serverless*: Serverless is about stop caring about your servers, but many implementations still require long YAML files to describe your infrastructure, and you really need to know what you're doing. True Serverless means that you don't even care about cloud configuration, Booster will figure it out for you based on the code you write.
+* *Convention over Configuration:* We prefer to provide  standardized highly-opinionated modules than highly-configurable ones. This helps us to keep your code small and follow the best security and structure practices when deploying your applications to the cloud. High consistency in your project and code structure also helps to abstract out most of the boilerplate.
+* *Don't Repeat Yourself (Extreme edition):* /The only code that matters is the one that makes your application different/. We push TypeScript structure and type system to the limit to avoid the need of writing repetitive code, like object-to-JSON serializations, API or database schemas or redundant architecture layers. 
+* *Self-documenting APIs* We adopted GraphQL because it's a self-documenting standard. You can grab ApolloClient and start using a Booster backend in a minute. We try to be as close as that with any other integration.
```

Comment:
I would omit the specific GraphQL client and I would just sugfest to grab any GraphQL client

Reply:
I partially agree, as the websockets protocol we've implemented is the one used by Apollo, but at the same time, here in the docs it doesn't matter...

Reply:
```suggestion
* *Self-documenting APIs* We adopted GraphQL because it's a self-documenting standard. You can grab a standard GraphQL client like [ApolloClient](https://github.com/apollographql/apollo-client) and start using a Booster backend in a minute.
```